packages/taiko-client/docs/swagger.yaml

```diff
@@ -0,0 +1,222 @@
+definitions:
+  big.Int:
+    type: object
+  softblocks.BuildSoftBlockRequestBody:
+    properties:
+      transactionBatch:
+        $ref: "#/definitions/softblocks.TransactionBatch"
+    type: object
+  softblocks.BuildSoftBlockResponseBody:
+    properties:
+      blockHeader:
+        $ref: "#/definitions/types.Header"
+    type: object
+  softblocks.RemoveSoftBlocksRequestBody:
+    properties:
+      newLastBlockId:
+        type: integer
+    type: object
+  softblocks.RemoveSoftBlocksResponseBody:
+    properties:
+      headsRemoved:
+        type: integer
+      lastBlockId:
+        type: integer
+      lastProposedBlockID:
+        type: integer
+    type: object
+  softblocks.SoftBlockParams:
+    properties:
+      anchorBlockID:
+        description: AnchorV2 parameters
+        type: integer
+      anchorStateRoot:
+        type: string
+      coinbase:
+        type: string
+      timestamp:
+        description: Block parameters
+        type: integer
+    type: object
+  softblocks.TransactionBatch:
+    properties:
+      batchId:
+        type: integer
+      batchType:
+        $ref: "#/definitions/softblocks.TransactionBatchMarker"
+      blockId:
+        type: integer
+      blockParams:
+        $ref: "#/definitions/softblocks.SoftBlockParams"
+      signature:
+        type: string
+      transactions:
+        type: string
```

Comment:
What is the format of transactions string? Is it zlib compressed as it's done for the `TaikoL1.proposeBlock`?

Reply:
yes.

Reply:
It would be nice to have it in the description 🙂

Reply:
Added in comments, try adding in swagger now as well.